pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -687,7 +689,7 @@ def _read_configuration():
     if args.meta_data is None:
         args.meta_data = []
 
-    if (not hasattr(args, 'services')) or args.services is None:
+    if getattr(args, 'services', None) is None:
         args.services = []
```

Comment:
Now I'm thinking we may go even more concise with:
```python
args.services = getattr(args, 'services', [])
```
What do you think?

Reply:
yes that looks exactly how the default value of `getattr` is supposed to be used like :+1:

Reply:
True, the semantic would not be the same for the `args.services is None` use case. But is that even an use case with `add_argument('--service', dest='services', action='append')`? I'm really wondering

Reply:
hmm only if the default is wrong, I suppose. if we checked that the default has a proper value and never `None` then it would be fine